battdat/postprocess/tagging.py

```diff
@@ -120,6 +121,20 @@ def enhance(self, df: pd.DataFrame):
         return df[['method']]
 
 
+class AddState(RawDataEnhancer):
+    """
+    Marks states in which battery is charging, discharging, or resting
+    """
+    column_names = ['current']
+
+    def enhance(self, data: pd.DataFrame) -> None:
+        if 'state' in data.columns:
```

Comment:
Could you remove this `if` statement? Let's handle the skip if already present at another level of the software to keep the amount of repeated code minimal